src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellPageRendererTracker.cs

```diff
@@ -461,7 +461,7 @@ public override CGRect Frame
 
 			public override void WillMoveToSuperview(UIView newSuper)
 			{
-				if (newSuper != null)
+				if (newSuper != null && newSuper.Bounds != CGRect.Empty)
```

Comment:
I feel dumb, but I have no idea how this fixes it. I see that now the container frame first checks for a zero rect when being added, but does it still update when the title container changes?

How did it work before? Was previously it always had a size, and in ios 16 it does not and then gets resized later?